### PR TITLE
Update XLA Pin

### PR DESCRIPTION
**Pull request by lsy323**

### CODE REVIEW

**Comment by zpcore** [comment]
The TPU ci [result](https://github.com/pytorch/xla/pull/7797/checks?check_run_id=28233277472) actually failed:
```
Step #4 - "run_e2e_tests": + TPU_LIBRARY_PATH=/usr/local/lib/python3.10/site-packages/torch_xla/lib/libtpu.so
Step #4 - "run_e2e_tests": + test/tpu/run_tests.sh
Step #4 - "run_e2e_tests": + python3 test/test_operations.py -v
Step #4 - "run_e2e_tests": Traceback (most recent call last):
Step #4 - "run_e2e_tests":   File "/src/pytorch/xla/test/test_operations.py", line 3107, in <module>
Step #4 - "run_e2e_tests": Fail to import hypothesis in common_utils, tests are not derandomized
Step #4 - "run_e2e_tests":     torch_xla._XLAC._xla_set_mat_mul_precision('highest')
Step #4 - "run_e2e_tests": AttributeError: module '_XLAC' has no attribute '_xla_set_mat_mul_precision'. Did you mean: '_xla_set_use_full_mat_mul_precision'?
```
But it is still showing as pass?

**Comment by lsy323** [comment]
> The TPU ci [result](https://github.com/pytorch/xla/pull/7797/checks?check_run_id=28233277472) actually failed:
> 
> ```
> Step #4 - "run_e2e_tests": + TPU_LIBRARY_PATH=/usr/local/lib/python3.10/site-packages/torch_xla/lib/libtpu.so
> Step #4 - "run_e2e_tests": + test/tpu/run_tests.sh
> Step #4 - "run_e2e_tests": + python3 test/test_operations.py -v
> Step #4 - "run_e2e_tests": Traceback (most recent call last):
> Step #4 - "run_e2e_tests":   File "/src/pytorch/xla/test/test_operations.py", line 3107, in <module>
> Step #4 - "run_e2e_tests": Fail to import hypothesis in common_utils, tests are not derandomized
> Step #4 - "run_e2e_tests":     torch_xla._XLAC._xla_set_mat_mul_precision('highest')
> Step #4 - "run_e2e_tests": AttributeError: module '_XLAC' has no attribute '_xla_set_mat_mul_precision'. Did you mean: '_xla_set_use_full_mat_mul_precision'?
> ```
> 
> But it is still showing as pass?

Thanks for catching it, seems the branch is out of sync with the master, rebase and run again.

File: setup.py [hunk 1]
```diff
@@ -64,9 +64,9 @@
 
 base_dir = os.path.dirname(os.path.abspath(__file__))
 
-_date = '20240726'
+_date = '20240801'
 _libtpu_version = f'0.1.dev{_date}'
-_libtpu_storage_path = f'https://storage.googleapis.com/libtpu-nightly-releases/wheels/libtpu-nightly/libtpu_nightly-{_libtpu_version}+default-py3-none-any.whl'
+_libtpu_storage_path = f'https://storage.googleapis.com/libtpu-nightly-releases/wheels/libtpu-nightly/libtpu_nightly-{_libtpu_version}+nightly-py3-none-any.whl'
 _jax_version = f'0.4.31.dev{_date}'
```
Comment (will-cromar) [inline setup.py:70]: Please double check the JAX minor version against their index: https://storage.googleapis.com/jax-releases/jax_nightly_releases.html

Comment (lsy323) [reply setup.py]: Thanks for catching it! Updated to `0.4.32`

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**